build/build.ps1

```diff
@@ -106,6 +106,9 @@ function Build {
     $CodecovProj = Join-Path $PSScriptRoot 'Codecov.proj'
     & $MsbuildExe $CodecovProj /m /nologo /clp:Summary /nodeReuse:$nodeReuse /warnaserror /v:diag /t:Codecov /p:Configuration=$configuration /p:UseCodecov=$useCodecov /p:NuGetPackageRoot=$NuGetPackageRoot $properties
   }
+
+  $GenerateDependentAssemblyVersionsProj = Join-Path $PSScriptRoot 'Scripts\GenerateDependentAssemblyVersionFile.targets'
+  & $MsbuildExe $GenerateDependentAssemblyVersionsProj /m /nologo /clp:Summary /nodeReuse:$nodeReuse /warnaserror /p:Configuration=$configuration
```

Comment:
can we ensure that this produces a binlog on ci? I love turning things into msbuild but I thought the whole point was better logging?

Reply:
It does in a followup, this is temporary while build.ps1 exists.